test/support/entity-samples.mts

```diff
@@ -5,6 +5,7 @@ const { UUID } = CommonDBTypes;
 export const entitySimple = {
   name: 'Simple',
   changelogDate: '20220129000100',
+  jpaMetamodelFiltering: true,
```

Comment:
Added this here to have both positive testing for sql option and negative testing for non-sql option